app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= "#{@mechanic.name}'s Page" %></h1>
+
+<%= "Years of experience: #{@mechanic.years_of_experience} "%></br></br>
+
+
+<h3> Currently working on: </h3>
+<% @mechanic.rides.each do |ride| %>
+
+  <%= ride.name %></br>
+
+  <% end %>
+<h3> Add a ride to workload: </h3>
+  <%= form_with url: "/mechanic_rides/", method: :create, data: { turbo: false} do |f| %>
```

Comment:
If you want to get really explicit with this `url`, you could do 
```ruby
url: "/mechanics/#{@mechanic.id}/rides
```
And then your `method` would be a `post`. But this looks like it works as well.